docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -0,0 +1,559 @@
+
+With the new configuration, the CLI prompts you to enter username and password securely by default. Commands in the new `zowe config` command group let you manage security for any option value.
+
+## Installing @next version
+
+To get started, install the Zowe CLI `@next` version from the online registry. You can follow this procedure for a first-time installation, or to update a currently installed version.
+
+**Follow these steps:**
+
+1. Meet the [software requirements for Zowe CLI](https://docs.zowe.org/stable/user-guide/systemrequirements.html#zowe-cli-requirements).
+
+2. To install or update the core CLI, open a command-line window and issue the following command:
+
+   ```
+   npm install -g @zowe/cli@next
+   ```
+
+3. Meet the [software requirements for each plug-in](https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html#software-requirements-for-zowe-cli-plug-ins).
+
+4. (Optional) Check [npmjs.com](https://www.npmjs.com/) for any Zowe plug-ins that have an `@next` version available. If an `@next` version is available, you can issue the following command (substituting the plugin name) to install it. If no `@next` version, try installing the `@latest` version of the plug-in:
+
+    ```
+    zowe plugins install @zowe/<plugin-name>@next
+    ```
+
+   Zowe CLI and optional plug-ins are installed!
+
+5. Open a command-line window and issue the following command:
+
+   ```
+   zowe scs revert --force
+   ```
+
+6. If you previously had the Secure Credential Store plug-in installed, uninstall it now to avoid unexpected behavior. Issue the following command:
+
+    ```
+    zowe plugins uninstall @zowe/secure-credential-store-for-zowe-cli
+    ```
+
+7. If you previously had an instance of Zowe CLI installed, your current configuration files are ignored if zowe.config.json is found globally, at the project level, or up the directory structure. Delete the following files from your local `.zowe/` directory:
+   - `.zowe/settings/imperative.json`
+   - `.zowe/profiles`
+
+   **Important!** Prior to deleting the contents of the `/profiles` directory, take note of any mainframe service details that you need (host, port, etc...). You might want to save the entire `/profiles` directory to another location on your computer so that you can reference or restore the profiles later.
+
+You can now configure the CLI and issue commands.
+## Initializing global configuration
+
+To begin, define a connection to z/OSMF and initialize your configuration files. We recommend this method for getting started, but you can choose create the configuration files manually if desired.
+
+**Follow these steps:**
+
+1. Issue the following command:
+
+   ```
+   zowe config init --global
+   ```
+
+   The CLI provides a series of prompts.
+
+2. Respond to the prompts to enter a username and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
```

Comment:
I think that we should replace "'--global' option" with "config init command"

Reply:
Agreed, from comment in #807:
> The --global flag is not related to secure credential storage.

Reply:
Updated